### PR TITLE
New C2 Chem for Brain Damage

### PR DESCRIPTION
**Pull request by sqnztb**
## About The Pull Request
Adds a new chem to treat brain damage: Proto-Neurite. Made from equal parts phenol, stable plasma, and sugar. It heals three brain damage, down to 30, while dealing 1 toxin and 1 heart damage per tick, and has a small chance to give a seizure.

## Why It's Good For The Game
After playing rounds on Manuel with a character who has Brain Tumor and Extreme Medicine Allergy, rolling a Mannitol allergy is great if you want to get to know your doctors in medbay. With brain death coming at sixteenish minutes, they need to do brain surgery every ten to fifteen minutes at most to keep you on this side of the grave. It's not so great if medbay is busy with patients who aren't total resource drains on the station. This adds a second chem to treat the brain damage. It does more raw brain healing than mannitol does with some pretty significant drawbacks. Unlike mannitol, it does not stop the brain tumor damage from ticking, nor will it completely heal the brain meaning that you can still accumulate traumas even if you had a thousand units of it in you (and some way of dealing with the toxin/heart damage). It is definitely not a chem to be used often, moreso just to keep you off death's door until medical can accommodate you.

## Changelog
:cl:
add: New C2 chem for brain damage. Proto-Neurite, made from phenol, stable plasma, and sugar.
/:cl:

### CODE REVIEW

**Comment by itseasytosee** [comment]
The more obvious solution to me is to buff neurine to allow it to heal brain damage. The thing about your chem is no one will use it in place of mannitol outside of the edge case scenario provided above as it is not easier to create and using it would not be less time consuming as you now alow have to deal with the toxin damage you have accumulated. The only other scenario I  could see this being used in is perhaps a combat chem when you are in combat that will result in brain damage but for whatever reason, that brain damage would pose more risks to you than the normal damage.
tldr; Its bloat.

**Comment by sqnztb** [comment]
> The more obvious solution to me is to buff neurine to allow it to heal brain damage. The thing about your chem is no one will use it in place of mannitol outside of the edge case scenario provided above as it is not easier to create and using it would not be less time consuming as you now alow have to deal with the toxin damage you have accumulated. The only other scenario I could see this being used in is perhaps a combat chem when you are in combat that will result in brain damage but for whatever reason, that brain damage would pose more risks to you than the normal damage. If you needed to make a new chem it should be useful outside of the scenario of I am allergic to mannitol and have a brain tumor.
> tldr; Its bloat.

Fair points. I will admit that my reasoning for this pull is biased, but I feel like I've tried to hold to the chems/reagents design goals.

I feel like the game needs a C2 brain chem simply because the brain cannot be replaced like other organs and damage to it must be treated. Regarding the creation of it, most of the C2 chems (helbital and seiver being the exceptions) require a couple of single chems and an assembled component/chem which is why the recipe is what it is--acetone + two singles. I originally considered had the brain healing at 5/tick (keeping in the idea of it being an "emergency" med) but thought that might be a little too good at healing the brain damage down. As it stands, 5u of this chem will heal 75 brain in exchange for 25 toxin and a net 23 heart by the time it metabolizes out. I think the 1/tox+heart per tick is an acceptable tradeoff, but these could be dialed back to be fractional similar to libital's 0.3 liver or aiuiri's 0.25 eye. The seizure addition may be a bit overkill but it adds some uniqueness not found in other chems. 

And for what it's worth, the other time I could see this being useful is when someone ingests omega weed (or ODs on bath salts, really) when you could really need some extra brain heals. Admittedly this could also be considered an edge case.

I did consider the thought of buffing neurine, but at that point what purpose does mannitol serve? Relegated to sidelines save for the occasional time where you have to pour some on a brain before shoving it in an MMI?

**Comment by Tlaltecuhtli** [comment]
meth, bath salts and other high brain dmg things are based on the fact that outside of mannitol you need exotic things to have a passive brain heal, like nanites or virus or brain folding surgery

**Comment by Jared-Fogle** [comment]
Tlal is right, though why are you playing a character with a *brain tumor* and *extreme medicine allergies* and are surprised that you're spending all your time in medbay?

**Comment by Winterous** [comment]
Ignoring issues others have raised, I think it's probably reasonable to cap the toxin damage this does, maybe to 20-30, so it just makes you sickly, not potentially dead. Also 1 organ damage per tick is quite a lot, maybe tone that down.

**Comment by sqnztb** [comment]
> Tlal is right, though why are you playing a character with a _brain tumor_ and _extreme medicine allergies_ and are surprised that you're spending all your time in medbay?
> 
> Also, did you know you can give epinephrine to someone to stop allergic reactions? 😄

Roleplay, mostly. And to remind doctors that crews' medical records matter sometimes. I also think it's hilarious to yell at docs for not being thorough before thanking them for keeping me sort of alive. Believe it or not, this combo generally doesn't keep me in medbay any more time than either of these would individually. It's only the mannitol allergy RNG that causes problems. I'm either getting constant surgery or essentially round removed at the sixteen minute mark. Re: Epinephrine, yeah. It stops the metabolization, and when the epinephrine wears off you get the histamine ride. Sanguirite allergy has caused me (and by me, I mean medbay) many more problems because people are super quick to jab a body with an epipen, fix it up, and then wonder why you've died again as soon as you're defibbed.

>Ignoring issues others have raised, I think it's probably reasonable to cap the toxin damage this does, maybe to 20-30, so it just makes you sickly, not potentially dead. Also 1 organ damage per tick is quite a lot, maybe tone that down.

My original implementation actually did cap the toxin damage. It did 3 tox/tick up to 30, but I wasn't sure if that would actually fly with the category 2 design.

`// Category 2 medicines are medicines that have an ill effect regardless of volume/OD to dissuade doping. Mostly used as emergency chemicals OR to convert damage (and heal a bit in the process). The type is used to prompt borgs that the medicine is harmful.`

If the damage is capped then it goes against this design goal, does it not? The heart damage is still there, true, but if you space out your dosing enough to get the maximum amount of brain healing out of each dose, i.e. not healing the brain down below the cutoff, you're only taking a net ~6 heart damage every time.

**Comment by wesoda25** [comment]
If you have a brain tumor and allergy to mannitol just take the L

**Comment by GoldenAlpharex** [comment]
I mean, another way for this to not be a problem would be, you know, to remove Mannitol from the possible medicine allergy list. Or just to stop having a combo of those two quirks. I just think that, in general, the motivation behind this PR is a little lacking and is too much of an edge case. It's going to end up as another one of those chemicals nobody knows how to make, because in the end it's not really worth it when there's better alternatives out there.

**Comment by optimumtact** [comment]
You went out of your way to inconvenience yourself and then decided to weaken gameplay for everyone else to suit yourself
